x/incentive/keeper/grpc_query.go

```diff
@@ -0,0 +1,157 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/umee-network/umee/v4/x/incentive"
+)
+
+var _ incentive.QueryServer = Querier{}
+
+// Querier implements a QueryServer for the x/incentive module.
+type Querier struct {
+	Keeper
+}
+
+func NewQuerier(k Keeper) Querier {
+	return Querier{Keeper: k}
+}
+
+func (q Querier) Params(
+	goCtx context.Context,
+	req *incentive.QueryParams,
+) (*incentive.QueryParamsResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
+	}
+
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	params := q.Keeper.GetParams(ctx)
+
+	return &incentive.QueryParamsResponse{Params: params}, nil
+}
+
+func (q Querier) IncentiveProgram(
+	_ context.Context,
+	req *incentive.QueryIncentiveProgram,
+) (*incentive.QueryIncentiveProgramResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
+	}
+
+	program := incentive.IncentiveProgram{}
+
+	resp := &incentive.QueryIncentiveProgramResponse{
+		Program: program,
+	}
+
+	return resp, incentive.ErrNotImplemented
```

Comment:
let's remove the `incentive.ErrNotImplemented` error and use one of the standard grpc `status` codes or `sdkerrors`.

Reply:
I'll remove that error in the next few PRs (it's only a "TODO" type error during development)